config/locales/devise/en.yml

```diff
@@ -150,5 +154,6 @@ en:
         confirm_with_voice: Confirm with voice call
       otp_method:
         title: How would you like to receive your passcode?
+        instruction: You can change your choice at any time
```

Comment:
All good points. However, I can easily see how we might be more precise without destroying plain language simplicity. Let's go with this for `otp_method.instruction`:

`You can change your choice the next time you sign in`

cc @jeanninehunter